pkg/registry/apis/folders/register.go

```diff
@@ -189,8 +189,10 @@ func (b *FolderAPIBuilder) GetAuthorizer() authorizer.Authorizer {
 func authorizerFunc(ctx context.Context, attr authorizer.Attributes) (*authorizerParams, error) {
 	verb := attr.GetVerb()
 	name := attr.GetName()
-	if (!attr.IsResourceRequest()) || (name == "" && verb != utils.VerbCreate) {
-		return nil, errNoResource
+	if verb != utils.VerbList {
```

Comment:
Wondering if we should still be checking that it's a resource request here:

```suggestion
    if (!attr.IsResourceRequest()) || (verb != utils.VerbList) {
```